br/pkg/storage/writer_test.go

```diff
@@ -102,8 +102,15 @@ func TestCompressReaderWriter(t *testing.T) {
 		ctx := context.Background()
 		storage, err := Create(ctx, backend, true)
 		require.NoError(t, err)
-		storage = WithCompression(storage, Gzip)
-		fileName := strings.ReplaceAll(test.name, " ", "-") + ".txt.gz"
+		storage = WithCompression(storage, test.compressType)
+		var suffix string
+		if test.compressType == Gzip {
```

Comment:
How about implementing a `String()` function for these compressType?

Reply:
Modified